examples/mosa/lib.py

```diff
@@ -23,7 +23,7 @@
 BASE_DIR = Path("/glade/campaign/mmm/c3we/prein/SouthAmerica/MCS-Tracking")
 """Base location on NCAR GLADE.
 
-Note that this location is only accessibly on Casper, not Hera!
+Note that this location is only accessibly on Casper, not Cheyenne(?)!
```

Comment:
```suggestion
Note that this location is only accessible on Casper, not Cheyenne
```
It was Cheyenne at the time we ran MOSA